src/main/java/uk/gov/hmcts/reform/userprofileapi/controller/UserProfileController.java

```diff
@@ -101,11 +105,16 @@
     )
     @ResponseBody
     public ResponseEntity<UserProfileCreationResponse> createUserProfile(@Valid @RequestBody UserProfileCreationData userProfileCreationData) {
-        //Creating new User Profile
 
+        UserProfileCreationResponse resource = null;
         validateCreateUserProfileRequest(userProfileCreationData);
 
-        UserProfileCreationResponse resource = userProfileService.create(userProfileCreationData);
+        if (userProfileCreationData.isResendInvite()) {
```

Comment:
there should be a test to  test this if else flow?

Reply:
It will be taken care by Integration test cases but unit test case also added.